src/freenet/client/filter/ElementInfo.java

```diff
@@ -206,8 +215,8 @@ public class ElementInfo {
 			"nth-last-child",
 			"nth-of-type",
 			"nth-last-of-type",
-			"link",
-			"visited",
+			"link", // inverse of visited, probably should be blocked?
```

Comment:
no, should not be blocked: this applies to any link. As long as visited is stripped out, there’s no privacy risk.

⇒ please adjust the comment.

Reply:
A future improvement to keep pages working could be to transparently rewrite "link" to "any-link". But that may be beyond the scope of this PR (no need to fix everything in one step).